### ISSUE TITLE
[Platform] GET kms_config fails (Cannot find universe ...)

### ISSUE THREAD

**Issue by SergeyPotachev** [yugabyte/yugabyte-db#7952]
By unknown reason it has fired only today while the diff has been submitted 12 days ago - commit is 1d975e4882f831b674f82ccbfa39c77faf3b6cf4 "[#7223, #7224] Platform: List associated universe with the KMS provider and certificates. (#7377)"

One of our portals had several records in `kms_history` with uuids for already removed universes. So we have today several exceptions like this:

2021-04-07 17:51:44,054 [ERROR] from application in application-akka.actor.default-dispatcher-312 - 

! @7jb24mbef - Internal server error, for (GET) [/api/v1/customers/11d78d93-1381-4d1d-8393-ba76f47ba7a6/kms_configs] ->
 
play.api.UnexpectedException: Unexpected exception[RuntimeException: Cannot find universe 
]
        at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:277)
        at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:204)
        at play.core.server.AkkaHttpServer$$anonfun$1.applyOrElse(AkkaHttpServer.scala:434)
        at play.core.server.AkkaHttpServer$$anonfun$1.applyOrElse(AkkaHttpServer.scala:432)
        at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
        at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
        at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:92)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
        at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
        at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:92)
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:49)
        at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.lang.RuntimeException: Cannot find universe 94b1a54d-7a7a-4ece-9aba-05d872cf11d4
        at com.yugabyte.yw.models.Universe.get(Universe.java:324)
        at com.yugabyte.yw.models.Universe.get(Universe.java:349)
        at com.yugabyte.yw.models.KmsHistory.getUniverses(KmsHistory.java:243)
        at com.yugabyte.yw.common.kms.util.EncryptionAtRestUtil.getUniverses(EncryptionAtRestUtil.java:211)
        at com.yugabyte.yw.controllers.EncryptionAtRestController.lambda$listKMSConfigs$0(EncryptionAtRestController.java:135)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.ArrayList$Itr.forEachRemaining(ArrayList.java:901)
        at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566)
        at com.yugabyte.yw.controllers.EncryptionAtRestController.listKMSConfigs(EncryptionAtRestController.java:144)
        at v1.Routes$$anonfun$routes$1.$anonfun$applyOrElse$90(Routes.scala:3730)

We need a migration to remove such records + cleanup of these records when universes are destroyed.

**Comment by SergeyPotachev** [comment]
The UI part will be fixed as #7950.  cc @yb-andrew

**Comment by sb-yb** [comment]
```
Caused by: java.lang.RuntimeException: Cannot find universe 435d5d0f-c0ca-4c9e-af3d-4bd0088e759a
        at com.yugabyte.yw.models.Universe.get(Universe.java:324)
        at com.yugabyte.yw.models.Universe.get(Universe.java:349)
        at com.yugabyte.yw.models.KmsHistory.getUniverses(KmsHistory.java:243)
        ...
```

Batch Universe lookup method `Universe.get(uuidSet)` method fails fast if any of the uuid is not found. 
Changing this to best effort and renaming `getAllPresent` should make server code more resilient to lack of referential integrity (kms config points to a universe id that is no longer active)
